### ISSUE TITLE
Shift content from "ABCD-J" tab to dataset properties

### ISSUE THREAD

**Issue by tmheunis** [abcd-j/data-catalog#16]
The extra "ABCD-J" tab is confusing. What should the user expect it to contain? In some cases there's useful info that would make more sense to show on the main dataset page next to the description, keywords, etc. For example, the superdataset's ABCD-J tab contains only a link to the abcd-j homepage, which would be useful to have visible immediately when viewing that dataset. Perhaps we should consider shifting this content?

**Comment by jsheunis** [comment]
Makes sense to me. This is a remnant from changes that were made for SFB1451, where I think a separate tab is perhaps more sensible since the projects in that consortium all involved some form of motor control, and hence presenting metadata from common categories in a single tab maybe made sense. Or it could just be that we didn't have a better technical solution than putting that info into a new tab (`additional_display` in the schema). But I agree some of these fields should rather be displayed "out in the open" rather than hidden in a tab.
